sections/subsections/controlflow2.tex

```diff
@@ -79,9 +88,57 @@ \subsection{if, if-else and ?:} \todo{Jeg blander meget rundt i statement og con
 
 An example of the ternary operator \texttt{?:} is shown in \myref{terlst}. 
 If the boolean statement before the question mark is true, then the first statement is executed otherwise the second is. 
-The ternary operator can also be used for an assignment. 
+The ternary operator can also be used for an assignment, only if the value getting assigned to is left of the question mark.
+As the ternary operator allows any statement in its middle body, it is possible to chain them together, just like a if-else chain. 
+
+\subsection{while}
+An example of the while loop in GAMBLE is shown in \myref{whilelst}. 
+The syntax is identical to the if statement, which also makes it look like a function in certain ways. 
+It's also identical to the syntax found in many other programming languages and is therefore similar to many programmers. \todo{Skal sådan et argument være med?}

```

Comment:
Hvilken syntax referere vi til? funktioner, if eller while loop?
"The while syntax is also identical [...]"
Desuden skal It is skrives ud, ingen semikolon i skriftsprog